frontend/src/v5/ui/routes/dashboard/projects/federations/federationsList/federationsList.component.tsx

```diff
@@ -0,0 +1,172 @@
+	hasFederations,
+}: IFederationsList): JSX.Element => {
+	const { teamspace, project } = useParams() as { teamspace: string, project: string };
+
+	const { sortedList, setSortConfig } = useOrderedList(federations, DEFAULT_SORT_CONFIG);
+	const {
+		setSearchInput,
+		searchInput,
+		filterQuery,
+	} = useSearchInput(search);
+	const isListPending = FederationsHooksSelectors.selectIsListPending();
+	const areStatsPending = FederationsHooksSelectors.selectAreStatsPending();
+
+	const setFavourite = (id: string, value: boolean) => {
+		if (value) {
+			FederationsActionsDispatchers.addFavourite(teamspace, project, id);
+		} else {
+			FederationsActionsDispatchers.removeFavourite(teamspace, project, id);
+		}
+	};
+
+	return (
+		<Container>
+			<DashboardListCollapse
+				title={<>{title} {!isListPending && `(${federations.length})`}</>}
+				tooltipTitles={titleTooltips}
+				isLoading={areStatsPending}
+				sideElement={(
+					<CollapseSideElementGroup>
+						<Trans
+							id="federations.search.placeholder"
+							message="Search..."
+							render={({ translation }) => (
+								<SearchInput
+									onClear={() => setSearchInput('')}
+									onChange={(event) => setSearchInput(event.currentTarget.value)}
+									value={searchInput}
+									placeholder={translation as string}
+								/>
+							)}
+						/>
+						<Button
+							startIcon={<AddCircleIcon />}
+							variant="contained"
+							color="primary"
+						>
+							<Trans id="federations.newFederation" message="New Federation" />
+						</Button>
+					</CollapseSideElementGroup>
+				)}
+			>
+				<DashboardListHeader onSortingChange={setSortConfig} defaultSortConfig={DEFAULT_SORT_CONFIG}>
+					<DashboardListHeaderLabel name="name">
+						<Trans id="federations.list.header.federation" message="Federation" />
+					</DashboardListHeaderLabel>
+					<DashboardListHeaderLabel name="issues" width={165}>
+						<Trans id="federations.list.header.issues" message="Open issues" />
+					</DashboardListHeaderLabel>
+					<DashboardListHeaderLabel name="risks" width={165}>
+						<Trans id="federations.list.header.risks" message="Open risks" />
+					</DashboardListHeaderLabel>
+					<DashboardListHeaderLabel name="containers" width={165}>
+						<Trans id="federations.list.header.containers" message="Containers" />
+					</DashboardListHeaderLabel>
+					<DashboardListHeaderLabel name="category" width={188}>
+						<Trans id="federations.list.header.category" message="Category" />
+					</DashboardListHeaderLabel>
+					<DashboardListHeaderLabel name="lastUpdated" width={180}>
+						<Trans id="federations.list.header.lastUpdated" message="Last updated" />
+					</DashboardListHeaderLabel>
+				</DashboardListHeader>
+				{useMemo(() => (
+					<DashboardList>
+						{!isEmpty(sortedList) ? (
+							sortedList.map((federation, index) => (
+								federation.hasStatsPending ? (
+									<SkeletonListItem delay={index / 10} key={federation._id} />
```

Comment:
I think we could have this inside of fedeationListItem, as it is it responsability to represent a ui item of a particular federation, including if it is fully loaded. Same goes with containers. If you think this doesnt seem like a good idea, let me know.